core/trino-spi/src/main/java/io/trino/spi/StandardErrorCode.java

```diff
@@ -152,6 +152,7 @@ public enum StandardErrorCode
     MISSING_PATH_NAME(128, USER_ERROR),
     INVALID_PLAN(129, USER_ERROR),
     INVALID_VIEW_PROPERTY(130, USER_ERROR),
+    INVALID_ENTITY_KIND(131, USER_ERROR),
```

Comment:
> Add StandardErrorCode.ILLEGAL_ENTITY_KIND

The commit message is wrong though it's too late.

Reply:
What is wrong with the commit message?

Reply:
The commit added `INVALID_ENTITY_KIND`, not `ILLEGAL_ENTITY_KIND`.

Reply:
The commit message says "illegal" while the error code const says "invalid". I guess that what's Yuya meant

Reply:
I didn't notice that and it's my fault